detectable/src/test/java/com/synopsys/integration/detectable/detectables/bazel/functional/bazel/pipeline/PipelineTest.java

```diff
@@ -104,35 +104,97 @@
                                                    + "  deps = [],\n"
                                                    + ")\n";
 
+    private static final String CQUERY_OUTPUT_MIXED_TAGS = "java_import(\n"
```

Comment:
I think we should make a utility method to create this so we don't have the newlines and the + signs. 

`private static final String CQUERY_OUTPUT_MIXED_TAGS = createOutput("java_import(",
                                                        "  name = \"sso-adminsdk\",",
                                                        "  tags = [\"__SOME_OTHER_TAG__\"],",
                                                        "  generator_name = \"sso-adminsdk\",",
`